README.md

```diff
@@ -84,6 +100,77 @@ Load the location from your order
           say_template:
           - "Today in {{ location }} the weather is {{ weather_today }} with a temperature of {{ temp_today_temp }} degree and tomorrow the weather will be {{ weather_tomorrow }} with a temperature of {{ temp_tomorrow_temp }} degree"
           
+```
+Forecast example for English users:
+```
+  - name: "weather-forecast-daily"
+    signals:
+      - order: "How is the weather on {{ day }} in {{ location }}"
+    neurons:
+      - openweathermap:
+          api_key: "fdfba4097c318aed7836b2a85a6a05ef"
+          lang: "en"
+          location: "{{ location }}"
+          day: "{{ day }}"
+          say_template: "The wheater on {{ forecast_day }} will be {{ forecast_weather_descripton }} with temperatures from {{ forecast_min_temp }} to{{ forecast_max_temp }} degree"
+```
+Forecast eample for French users:
+```
+  - name: "weather-forecast-daily"
+    signals:
+      - order: "Quelle est la météo {{ day }} sur {{ location }}"
+    neurons:
+      - openweathermap:
+          api_key: "fdfba4097c318aed7836b2a85a6a05ef"
+          lang: "fr"
+          country: "FR"
+          location: "{{ location }}"
+          day: "{{ day }}"
+          days:
+            Monday: "Lundi"
+            Tuesday: "Mardi"
+            Wednesday: "Mercredi"
+            Thursday: "Jeudi"
+            Friday: "Vendredi"
+            Saturday: "Samedi"
+            Sunday: "Dimanche"
+          say_template: "La météo à {{ forecast_day }} sera {{ forecast_weather_descripton }} avec des températures allant de {{ forecast_min_temp }} à {{ forecast_max_temp }} degrés."
+```
+Forecast for German users:
+```
+  - name: "weather-forecast-daily"
+    signals:
+      - order: "wie wird das wetter am {{ day }}"
+    neurons:
+      - openweathermap:
+          api_key: "fdfba4097c318aed7836b2a85a6a05ef"
+          lang: "de"
+          country: "DE"
+          location: "{{ location }}"
+          day: "{{ day }}"
+          days:
+            Monday: "Montag"
```

Comment:
case sensitive? If yes, need to be documented

Reply:
Its a dict, and it will be iterate through it with .lower()